locales/de.json

```diff
@@ -247,25 +247,25 @@
   "app.auctions.search.option.sort.priceAsc": "Preis (aufsteigend)",
   "app.auctions.search.option.sort.priceDesc": "Preis (absteigend)",
   "app.auctions.search.nemesis.label.quirk": "Marotte",
-  "app.auctions.search.nemesis.placeholder.quirk": "Any",
+  "app.auctions.search.nemesis.placeholder.quirk": "Beliebig",
   "app.auctions.search.nemesis.label.element": "Element",
   "app.auctions.search.nemesis.label.damage": "Beschädigung",
   "app.auctions.search.nemesis.label.ephemera": "Ephemera",
-  "app.auctions.search.nemesis.placeholder.ephemera.any": "Any",
+  "app.auctions.search.nemesis.placeholder.ephemera.any": "Beliebig",
   "app.auctions.search.nemesis.placeholder.ephemera.without": "Ohne",
   "app.auctions.search.nemesis.label.havingEphemera": "Mit Ephemera",
-  "app.auctions.search.nemesis.label.weaponName": "Weapon Name",
-  "app.auctions.search.nemesis.placeholder.weaponName": "Any",
+  "app.auctions.search.nemesis.label.weaponName": "Name der Waffe",
+  "app.auctions.search.nemesis.placeholder.weaponName": "Auswählen",
```

Comment:
This is the only "Any" I decided to translate differently. I feel like "Auswählen" feels natural in a mandatory dropdown menu.